docs/README.md

```diff
@@ -15,15 +15,21 @@ _Example of a quick order page without customization_
 
 ## Configuration
 
-Add the Quickorder app to your theme's dependencies in `manifest.json`. For example:
+### Environment instalation
+
+Install the app by running `vtex install vtex.quickorder` on the [vtex toolbelt](https://vtex.io/docs/concepts/toolbelt/)
+
+### Store Theme: Dependencies
+
+After installing the app in your account, add it as a `peerDependency` in your `store-theme`:
```

Comment:
I would move this part down to the "Advanced configurations" section, as it is only necessary to add the app to the peerDependencies if the user wishes to modify the default block template. (If I understand correctly)